### DISCUSSION TITLE
Add a feature that can add text to images using selected ttf fonts?

### DISCUSSION THREAD

**Discussion by Emperormummy** [T8RIN/ImageToolbox#771] [Ideas]
Add a feature that can add text to images using selected ttf fonts?

**Comment by T8RIN** [comment]
Yes, it's planned, but as for now, you can use the watermarks feature to achieve this

**Comment by Dark6id8SSS** [comment]
A huge request, please add the option to select a file (s) from the Explorer (Directory on directories and folders). It is difficult to search for the desired file among thousands of other files.
Quote:)) 😃Thank You So Much! Helped!👍

**Reply by T8RIN** [reply]
It's possible a long time ago, just choose the needed image source in the settings

**Reply by Emperormummy** [reply]
> It's possible a long time ago, just choose the needed image source in the settings 

I now feel that this feature is not particularly necessary, using the app's internal provision of several fonts is also sufficient to use, the main function is to add the text can be able to freely place the position on the pictures. Not just the several arrangements of watermarks

**Reply by T8RIN** [reply]
You can use a text brush in the image drawr
